modules/transport-netty4/src/test/java/org/elasticsearch/rest/Netty4HeadBodyIsEmptyIT.java

```diff
@@ -62,9 +62,12 @@ private void createTestDoc(final String indexName, final String typeName) throws
 
     public void testDocumentExists() throws IOException {
         createTestDoc();
-        headTestCase("/test/test/1", emptyMap(), greaterThan(0));
-        headTestCase("/test/test/1", singletonMap("pretty", "true"), greaterThan(0));
-        headTestCase("/test/test/2", emptyMap(), NOT_FOUND.getStatus(), greaterThan(0));
+        headTestCase("/test/test/1", emptyMap(), OK.getStatus(), greaterThan(0),
```

Comment:
Maybe we could switch to the `_doc` type in this test, so that we don't need to check deprecation warnings for get requests. The main purpose of the tests is to check how `HEAD` is handled, and not verify deprecation warnings (we do that in other places), so it could be nice to bring the test in line with the new API.